wolfcrypt/benchmark/benchmark.c

```diff
@@ -4986,10 +4986,14 @@ void bench_ed25519KeySign(void)
     /* declared above at line 239 */
     /* extern   double current_time(int reset); */
 
-#elif defined FREERTOS
+#elif defined(FREERTOS) || defined(WOLFSSL_ESPIDF)
```

Comment:
I think "task.h" is a FreeRTOS header, and not specific to the ESP-IDF.  It looks like your settings.h automatically defines `FREERTOS` when `WOLFSSL_ESPIDF` is defined.  This probably means we can remove the check here for `WOLFSSL_ESPIDF` right?

Reply:
Yes, right. I am going to remove the ESP-IDF.

Reply:
Thanks